### ISSUE TITLE
[Suggestion] Silktouch addon for Block Destroyer/Laser

### ISSUE THREAD

**Issue by ffuentesm** [Rearth/Oritech#201]
### Describe the feature you'd like

Since there is already a "fortune Addon" with the Machine Yield Addon. I think it would be beneficial to have a Silktouch variant for the Block Destroyer and Laser. 
This will help with Quaries and other farms that would prefer to extract blocks without damaging them. 

### Additional context

I was planning on making an ice far and using the block destroyer to extract the ice for me, but the way things are now, it just breaks the block without picking it up.

**Comment by jshipley** [comment]
I agree, this would be very useful. It would be great for ice (especially with ice being used for reactor cooling), but also good for mining stone or ores.

Any ideas for what the addon should look like?

**Comment by The4codeblocks** [comment]
> Any ideas for what the addon should look like?

some kind of saw, to cut around the block into a cube
